### ISSUE TITLE
Option to not have a pop-up for well known terms

### ISSUE THREAD

**Issue by yannayp** [jzohrab/lute-v3#348]
**Is your feature request related to a problem? Please describe.**

I've been using Lute to learn Portuguese from a very early stage of my journey with the language. Many basic words are now marked as Well Known, but they keep popping up when hovering over a text, which can be distracting and ultimately unnecessary (as they are well known).

**Describe the solution you'd like**

An ideal solution would be an option to turn off pop-ups when hovering over Well Known (and Ignored) terms. Of course, it should be optional and be off by default, probably.

**Describe alternatives you've considered**

Maybe this feature should be built-in for the "Focus mode"?

**Additional context**

---

**Comment by jzohrab** [comment]
Silly question, but could you just ... not hover over them? :-)  I understand what you're saying, just interested in your motivation for the question.  Thanks for the issue!

**Comment by yannayp** [comment]
Yeah, of course, I try not to hover too much. It's just a habit that a lot of people have when reading on the computer, to follow along with the cursor. And I think it makes sense as a feature, but obviously not top priority.

**Comment by Mycheze** [comment]
I second this option. I think it'd be a good idea for it to be hidden in the settings, since for most people, default behavior is good.

I think that there should just be a delay option for each learning level. When I hover over a 1-2, I want to see it right away, but with a 3 or 4, I sometimes hover over without really meaning it and see the word instantly when I'd rather think for a second. For ignored words, I probably always want to be able to see them (if there's anything there) and then known words I would want like a 3 second delay before seeing the translation.

Yannayp is right that it's just a habit to follow along with what I'm reading. And a little delay would mean that I could do that without switching my brain into "just read the English" mode.

**Comment by jzohrab** [comment]
> I think that there should just be a delay option for each learning level.

Avoid word "just" because it trivializes how tough things can be to do! :-)   (insert El Risitas picture here)  But it's a nice idea.

I hear you on the switching brain thing.  For an initial step I think a setting to say "only show the popup for this status or lower" would do.

**Comment by Mycheze** [comment]
Oh yeah, I try to remove "just" wherever I can. In this case, it was more in response to the other option of having it not appear. I don't think that's needed, "just" a delay would be fantastic. But my post would have been just fine without the word just 🙏 Thanks for the reminder!
